rust/src/delta.rs

```diff
@@ -1108,9 +1115,14 @@ impl DeltaTable {
             "timestamp".to_string(),
             Value::Number(serde_json::Number::from(Utc::now().timestamp_millis())),
         );
+        let mut commit_info = Map::<String, Value>::new();
+        commit_info.insert(
+            "commitInfo".to_string(),
```

Comment:
I think this would result in `{"commitInfo": "commitInfo": {...}}` when we serialize `Action::commitInfo(commit_info)` to JSON string?

Reply:
Good catch, let me fix 😃